compileopts/target.go

```diff
@@ -272,6 +276,7 @@ func defaultTarget(goos, goarch, triple string) (*TargetSpec, error) {
 		GDB:         "gdb",
 		PortReset:   "false",
 		FlashMethod: "native",
+		CodeModel:   "default",
```

Comment:
You could leave this out because an empty code model is already equivalent to "default" (to reduce clutter).